src/main/java/seedu/address/model/person/Money.java

```diff
@@ -41,6 +47,39 @@ public static boolean isValidMoney(Integer amount) {
         return amount >= 0;
     }
 
+    /**
+     * Computes the sum of a predefined value and the current value.
+     *
+     * @param amountToAdd the money to be added.
+     * @return Money that consists of the sum of the 2 amounts.
+     * @throws CommandException if an error occurs during addition.
+     */
+    public Money addTo(Money amountToAdd) throws CommandException {
+        try {
+            Integer amountSum = Math.addExact(this.value, amountToAdd.value);
+            return new Money(amountSum);
+        } catch (ArithmeticException e) {
+            throw new CommandException(MESSAGE_AMOUNT_TOO_LARGE);
+        }
+    }
+
+    /**
+     * Computes the subtraction of a predefined value from the current value.
+     *
+     * @param amountToSubtract the money to be subtracted.
+     * @return Money that consists of amount after subtraction.
+     * @throws CommandException if the final value is negative.
+     */
+    public Money subtract(Money amountToSubtract) throws CommandException {
+        Integer finalAmount = this.value - amountToSubtract.value;
+
+        if (finalAmount < 0) {
+            throw new CommandException(MESSAGE_NEGATIVE_AMOUNT);
```

Comment:
Is this intended as an example of defensive coding? The negative number would not be able to pass the parsing in the first place. (as enforced by `Money::isValidMoney`)

Reply:
> Is this intended as an example of defensive coding? The negative number would not be able to pass the parsing in the first place. (as enforced by `Money::isValidMoney`)

Good catch! Forgot about Money checking whether it is negative. Have implemented a new code.